documentation/overview/account_setup/registration-and-login.md

```diff
@@ -3,89 +3,57 @@ ospool:
   path: overview/account_setup/registration-and-login.md
 ---
 
-Registration and Login for OSG Connect 
+Registration and Login for OSPool Access
```

Comment:
```suggestion
Getting Started: Overview of Requesting OSPool Access
```

Reply:
`Registration and Login for OSPool Access` is too similar to `Join and Log In to "OSG Connect" Access Points` - I'd like it to be more clear that this is the TRUE getting started point/guide